src/assembler/encoder.ts

```diff
@@ -350,7 +354,9 @@ function writeCodeInstruction(
  */
 function writeLiteralPool(writer: FileWriter, pool: ILiteralPool) {
   if (pool.entries.length > 0) {
-    writer.writeBytes(END_OF_CODE.toBytes(), 2)
+    writer.align(2)
+    writer.writeBytes(END_OF_CODE.toBytes())
+    writer.align(4)
```

Comment:
perhaps update comment so why is separated by end of code and aligned befor literal pool itself

Reply:
Comments have been improved.